streams/shared/src/main/scala/zio/stream/ZSink.scala

```diff
@@ -838,4 +838,28 @@ object ZSink extends ZSinkPlatformSpecificConstructors {
    */
   def sum[A](implicit A: Numeric[A]): ZSink[Any, Nothing, A, Nothing, A] =
     foldLeft(A.zero)(A.plus)
+
+  /**
+   * A sink that offers values to a given queue. Shuts down the queue if specified.
+   */
+  def toQueue[R, I](
```

Comment:
```suggestion
  def fromQueue[R, I](
```

Reply:
Then tests for this functions must be under `suite("nstructors"){}`